packages/react-transport-dom-webpack/src/ReactFlightWebpackPlugin.js

```diff
@@ -7,7 +7,34 @@
  * @flow
  */
 
+import {mkdirSync, writeFileSync} from 'fs';
+import {dirname, resolve} from 'path';
+
 export default class ReactFlightWebpackPlugin {
   constructor(options: {isServer: boolean}) {}
-  apply(compiler: any) {}
+
+  apply(compiler: any) {
+    compiler.hooks.emit.tap('React Transport Plugin', compilation => {
+      const json = {};
+      compilation.chunks.forEach(chunk => {
```

Comment:
If I understand correctly, in webpack the same module might be duplicated between multiple chunks. Due to forEach here, the last set will win. I don't know what's a good way to pick but this seems like something to be worked out by the person productionizing this.

Reply:
The name is associated with the root/entry dependency, not the file. So we'll model the dependency which then points to a chunk. Webpack will decide for us where that dependency will point.